### DISCUSSION TITLE
How do I get xwindow to only display the app name?

### DISCUSSION THREAD

**Discussion by rew1nter** [polybar/polybar#2990] [Q&A] [answered]
```
[module/xwindow]
type = internal/xwindow
label = %title%
```
With the code above, this is the output of chrome when focused:
`Module: script · polybar/polybar Wiki - Google Chrome` 
Here I only want to show the `Google Chrome` part. Can this be done without scripting?

**Comment by rew1nter** [comment]
And if scripting is the only way, what should be the ideal interval for monitoring this window change activity?

**Reply by patrick96** [reply, marked as answer]
Scripting is the only way since you need to do some string manipulation. The interval really is a tradeoff between responsiveness and resource usage. I would say between 1 and 5 seconds is reasonable.
